{"uuid":"f844912a-fcc0-4d05-acdb-1852be6a42b6","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["2b13f0e8-a5f3-4c86-9cf8-848e29535d89"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884949796,"stop":1696884949796},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884949880,"stop":1696884949880},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884949963,"stop":1696884949963},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696884951990,"stop":1696884951991}],"start":1696884949500,"stop":1696884951991}